### PR TITLE
feat(e-z): new pages and logo update

### PR DESCRIPTION
**Pull request by tsoxas**
## Description 
Updated the logo and banner to use the new ones and added some pages that were previously missing

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
image
image
</details>

### CODE REVIEW

**Comment by ignSKRRRTT** [comment]
I love you tsoxas!!!! Big fan!

File: websites/E/e-z/presence.ts [hunk 1]
```diff
@@ -26,6 +26,39 @@ presence.on("UpdateData", async () => {
...
+					presenceData.state = "Viewing e-z.host's privacy policy";
+					break;
+
+				case "/tos":
+					presenceData.state = "Viewing e-z.host's terms of service";
+					break;
+
+				case "/resetpassword":
+					presenceData.state = "Resetting their password";
+					break;
```
Comment (EncryptedDev) [inline websites/E/e-z/presence.ts:48]: These aren't really needed

File: websites/E/e-z/presence.ts [hunk 2]
```diff
@@ -26,6 +26,39 @@ presence.on("UpdateData", async () => {
...
+				case "/reseller":
+					presenceData.state = "Viewing info about e-z.host's reseller";
+					break;
+
+				case "/oliver":
+					presenceData.state = "Looking at pictures of oliver";
+					break;
+
+				case "/uploadsystems":
+					presenceData.state = "Reading about a shit host";
```
Comment (EncryptedDev) [inline websites/E/e-z/presence.ts:59]: Explicit words are prohibited

File: websites/E/e-z/presence.ts [hunk 3]
```diff
@@ -26,6 +26,39 @@ presence.on("UpdateData", async () => {
...
+				case "/resetpassword":
+					presenceData.state = "Resetting their password";
+					break;
+
+				case "/reseller":
+					presenceData.state = "Viewing info about e-z.host's reseller";
+					break;
+
+				case "/oliver":
+					presenceData.state = "Looking at pictures of oliver";
```
Comment (EncryptedDev) [inline websites/E/e-z/presence.ts:55]: If "Oliver" is a proper noun, it should be given an upper case O

File: websites/E/e-z/metadata.json [hunk 4]
```diff
@@ -12,9 +12,9 @@
 		"e-z.host",
 		"e-z.bio"
 	],
-	"version": "1.0.0",
-	"logo": "https://i.imgur.com/Crfg6sn.png",
-	"thumbnail": "https://i.imgur.com/sm0hAr0.png",
+	"version": "1.0.1",
```
Comment (EncryptedDev) [inline websites/E/e-z/metadata.json:15]: Functionality was added so this should be a minor version bump instead

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/E/e-z/presence.ts (hunk 2 above)
Comment (EncryptedDev) [reply websites/E/e-z/presence.ts]: Still not an acceptable string

Comment (tsoxas) [reply websites/E/e-z/presence.ts]: Ok I completely removed it

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**